client/engine/store/mockstore.go

```diff
@@ -237,7 +242,11 @@ func (ms *MockStore) GetConsensusChannel(counterparty types.Address) (channel *c
 func (ms *MockStore) GetObjectiveByChannelId(channelId types.Destination) (protocols.Objective, bool) {
```

Comment:
Implementation performs some logic after obtaining the objective id.

Given the current code returns an objectiveId, may be better to just call directly into `ms.GetObjectiveById`

```
obj, err := ms.GetObjectiveById(protocols.ObjectiveId(id))
return obj, err == nil
```

(this compiles, runs, and passes, but highlights that the error return in `GetObjectiveByChannelId` should probably be enriched from an `ok` bool to a proper error)